app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,9 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+      <%= time_ago_in_words(quote.created_at || Time.now) %> ago
```

Comment:
This feels more like a workaround to me, but I like where your head is at. Can you try and fix the `production.sql` file so that we don't get any quotes without a null value for `created_at`